### ISSUE TITLE
Accessibility 

### ISSUE THREAD

**Issue by holtwick** [holtwick/onepile#134]
### Story

I'm blind and want to use OnePile. I use VoiceOver or any other screen reader. I would like to quickly navigate and find contents. Interacting with other apps for import and export should be easy (e.g. #133).

Sources:

- [Apple Accessibility for Developers](https://developer.apple.com/accessibility/)
- [WAI-ARIA Basics](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/WAI-ARIA_basics)
